src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -430,6 +432,71 @@ class spell_mage_cold_snap : public SpellScript
     }
 };
 
+// 153595 - Comet Storm (launch)
+class spell_mage_comet_storm : public SpellScript
+{
+    PrepareSpellScript(spell_mage_comet_storm);
+
+    class CometStormEvent : public BasicEvent
```

Comment:
Should the basic event not be outside the class?

Reply:
doesn't matter actually